.travis.yml

```diff
@@ -4,7 +4,10 @@ jdk:
   - oraclejdk8
   - openjdk7
 
-
+  
+before_install:
+        - sudo apt-get install libboost-dev libboost-test-dev
```

Comment:
The .travis.yml will be executed on a Travis VM, which is garanteed to be a debian derivate ;)
Having said this, the line is not actually needed, as the behave will only run 'sonar-runner' against checked in reports, no?
The boost header, libs, etc. are needed only for developing the tests, i.e. on your workstation. Not on the CI slave.

So not probs here

wont be executed on Windows